packages/react/src/bags/hooks/useBag.ts

```diff
@@ -159,7 +160,16 @@ const useBag = (options: UseBagOptions = {}) => {
   );
 
   useEffect(() => {
-    if (!isLoading && !isFetched && enableAutoFetch && userBagId) {
+    const updatedState = store.getState() as StoreState;
```

Comment:
Is this type not inferred by [getEntities](https://github.com/Farfetch/blackout/blob/main/packages/redux/src/entities/selectors/entity.ts#L43C5-L43C5)?
My concern is to avoid repeating `as StoreState` on every store.getState() call.

Reply:
`store.getState()` will return a value of the type that was passed to the `useStore()` call. As we are not passing any there it will assume as unknown, so we have to add a cast here. Both approaches are ok, but specifying the type on the `useStore()` might be cleaner but for this specific case we can leave as it is. What do you think?

Reply:
For sure. We can improve the generics in the future as well.